### PR TITLE
TRI-1149: adjust tavern.yml to set ticket-number for result export

### PR DESCRIPTION
**Pull request by ds-alexander-bulgakov**

### CODE REVIEW

File: .github/workflows/tavern.yml [hunk 1]
```diff
@@ -58,5 +61,4 @@ jobs:
           curl -H "Content-Type: multipart/form-data" \
           -u $JIRA_USERNAME:$JIRA_PASSWORD \
           -F 'file=@result.xml' \
-          "https://jira.catena-x.net/rest/raven/1.0/import/execution/junit?testExecKey=TRI-910"
-          
+          "https://jira.catena-x.net/rest/raven/1.0/import/execution/junit?testExecKey={github.event.inputs.execution-ticket || 'TRI-910'}"
```
Comment (ds-jhartmann) [inline .github/workflows/tavern.yml:64]: Please add `{github.event.inputs.execution-ticket || 'TRI-910'}` as env and use the env var in the curl:
env:
   TEST_EXECUTION_TICKET: ${{ github.event.inputs.execution-ticket || 'TRI-910' }}
...
run: | 
...
"https://jira.catena-x.net/rest/raven/1.0/import/execution/junit?testExecKey=$TEST_EXECUTION_TICKET

File: .github/workflows/tavern.yml [hunk 2]
```diff
@@ -36,7 +41,6 @@ jobs:
           pip install py
           pip install tavern
           pip list
-
```
Comment (ds-jhartmann) [inline .github/workflows/tavern.yml:39]: Please leave the blank lines between steps for better readability

File: .github/workflows/tavern.yml [hunk 3]
```diff
@@ -46,7 +50,6 @@ jobs:
           GLOBAL_ASSET_ID: ${{ github.event.inputs.global-asset-id || 'urn:uuid:513d7be8-e7e4-49f4-a22b-8cd31317e454' }}
         run: |
           python -m pytest api-tests/irs-api-tests.tavern.yaml --junitxml=tavern-results.xml
-
```
Comment (ds-jhartmann) [inline .github/workflows/tavern.yml:49]: Please leave the blank lines between steps for better readability

**Review by ds-jhartmann — CHANGES_REQUESTED** [review]

File: .github/workflows/tavern.yml (hunk 1 above)
Comment (ds-alexander-bulgakov) [reply .github/workflows/tavern.yml]: Changed, see new commit.

File: .github/workflows/tavern.yml (hunk 2 above)
Comment (ds-alexander-bulgakov) [reply .github/workflows/tavern.yml]: Blank line added, see new commit.

File: .github/workflows/tavern.yml (hunk 3 above)
Comment (ds-alexander-bulgakov) [reply .github/workflows/tavern.yml:49]: Blank line added, see new commit.

**Review by ds-jhartmann — APPROVED** [review]

**Pull request merged**